go.mod

```diff
@@ -28,18 +27,16 @@ require (
 	github.com/russross/blackfriday v2.0.0+incompatible
 	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
 	github.com/stretchr/testify v1.5.1
-	github.com/uncharted-distil/distil-compute v0.0.0-20200909155738-39c3281e4a53
+	github.com/uncharted-distil/distil-compute v0.0.0-20200917145207-f4b488934399
 	github.com/uncharted-distil/gdal v0.0.0-20200504224203-25f2e6a0dc2a
 	github.com/unchartedsoftware/plog v0.0.0-20170413154239-34d2bbd3c0a9
-	github.com/unchartedsoftware/witch v0.0.0-20200617171400-4f405404126f // indirect
```

Comment:
@cdbethune should I be worried?
This branch works fine when I run `make watch`.